aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/container/Place.java

```diff
@@ -24,11 +24,11 @@
 @Target({ElementType.FIELD, ElementType.METHOD})
 @Retention(RetentionPolicy.RUNTIME)
 @SuppressWarnings("unused")
-public @interface PlaceOn {
+public @interface Place {
     /**
-     * String equal to appropriate {@link AccordionPanel#title()} value or {@link Tab #title()}
+     * String equal to appropriate {@link AccordionPanel#title()} value or {@link Tab#title()}
```

Comment:
If specified, must contain the string that is equal to a {@link Tab} or an {@link AccordionPanel} title in order to place the current widget in the appropriate container. Skip this value if you don't need any specific placement